spec/features/loan_index_spec.rb

```diff
@@ -56,14 +60,28 @@
     context 'with no local translations' do
       before { @loans = create_list(:loan, 3, :with_foreign_translations, :featured) }
       it 'renders loan description with translation hint' do
-        visit public_loans_path
         click_link 'All'
         @loans.each do |loan|
           expect(page).to have_content loan.summary
         end
         expect(page).to have_selector '.loans_items span.translation.foreign_language', count: 3
       end
     end
+
+    context 'with divisions' do
+      scenario 'filters with division' do
+        select 'chicken', from: 'division'
+        @loans.each do |loan|
+          expect(page).to have_content loan.summary
+        end
+      end
```

Comment:
I don't see how this test works. The `@loans` were not assigned to the "chicken" division, were they? If they weren't, then they should *not* show when that division is selected. Either way, in order to test filtering, we need to test that some loans do show and also that some *don't* show. And the "don't show" expectation needs to come first, to make sure capybara waits for the filtering to happen.